scripts/util.py

```diff
@@ -207,25 +207,30 @@ def get_credentials() -> Optional[service_account.Credentials]:
 
 def create_client_options(api: ApiEndpoint = None) -> ClientOptions:
     """Create client options for cloud endpoints"""
-    ep = None
     ver = endpoint_version(api)
     ud = universe_domain()
-    if ud == DEFAULT_UNIVERSE_DOMAIN:
-        ud = None
+    co = ClientOptions()
+    if ud and ud != DEFAULT_UNIVERSE_DOMAIN:
+        try:
+            co.universe_domain = ud
+        except Exception:
+            log.error(
+                "Universe domain is not part of ClientOptions with installed version of google-api-core."
+            )
+            exit(1)
     if ver:
-        ep = f"https://{api.value}.{ud}/{ver}/"
+        co.api_endpoint = f"https://{api.value}.{ud}/{ver}/"
     log.debug(
```

Comment:
nit. You can simply log `co` object, it defines `__repr__` method:

```py
> co = ClientOptions(api_endpoint="https://compute.googleapis.com")
> print(f"{co}")
ClientOptions: {'api_endpoint': 'https://compute.googleapis.com/', 'client_cert_source': None, 'client_encrypted_cert_source': None, 'quota_project_id': None, 'credentials_file': None, 'scopes': None, 'api_key': None, 'api_audience': None}
```

Reply:
Done